test/cpp/tensorexpr/test_kernel.cpp

```diff
@@ -139,6 +139,9 @@ TEST(Kernel, _3) {
 }
 
 TEST(Kernel, _4) {
+  // disabled: doesn't do stride propagation, and isn't being used currently
```

Comment:
Maybe use `DISABLED_` feature of gtest instead? Also, from the comment and the test name (my fault, sorry), it's not clear *what* doesn't do stride propagation.